vyper/parser/stmt.py

```diff
@@ -441,27 +446,21 @@ def call(self):
         else:
             return external_call.make_external_call(self.stmt, self.context)
 
-    def parse_assert(self):
-
-        with self.context.assertion_scope():
-            test_expr = Expr.parse_value_expr(self.stmt.test, self.context)
-
-        if not self.is_bool_expr(test_expr):
-            raise TypeMismatchException('Only boolean expressions allowed', self.stmt.test)
-        if self.stmt.msg:
-            if not isinstance(self.stmt.msg, ast.Str):
-                raise StructureException(
-                    'Reason parameter of assert needs to be a literal string.', self.stmt.msg
-                )
-            if len(self.stmt.msg.s.strip()) == 0:
-                raise StructureException('Empty reason string not allowed.', self.stmt)
-            reason_str = self.stmt.msg.s.strip()
-            sig_placeholder = self.context.new_placeholder(BaseType(32))
-            arg_placeholder = self.context.new_placeholder(BaseType(32))
-            reason_str_type = ByteArrayType(len(reason_str))
-            placeholder_bytes = Expr(self.stmt.msg, self.context).lll_node
-            method_id = fourbytes_to_int(sha3(b"Error(string)")[:4])
-            assert_reason = [
+    def _assert_reason(self, test_expr, msg):
+        if not isinstance(msg, ast.Str):
+            raise StructureException(
+                    'Reason parameter of assert needs to be a literal string.',
+                    msg)
+        if len(msg.s.strip()) == 0:
+            raise StructureException('Empty reason string not allowed.',
+                    self.stmt)
```

Comment:
`vyper/parser/stmt.py:456:21: E128 continuation line under-indented for visual indent` is the linter exception. Does this even need to be on a new line?

Reply:
80 chars